packages/vaadin-login/test/login-overlay.test.js

```diff
@@ -64,7 +63,7 @@ describe('opened overlay', () => {
   });
 
   it('should not close on click outside', () => {
-    tap(overlay.$.vaadinLoginOverlayWrapper.$.backdrop);
+    click(overlay.$.vaadinLoginOverlayWrapper.$.backdrop);
```

Comment:
question: not clear why a `tap` is replaced by a `click`. was it intentional?

Reply:
Restored.